src/topology/uniform_space/compact.lean

```diff
@@ -197,20 +198,41 @@ begin
   exacts [mem_ball_self _ (hT a a.2), mem_Inter₂.1 h a ha],
 end
 
-lemma continuous.uniform_continuous_of_zero_at_infty {f : α → β} [has_zero β]
-  (h_cont : continuous f) (h_zero : tendsto f (cocompact α) (𝓝 0)) : uniform_continuous f :=
+@[to_additive]
+lemma continuous.uniform_continuous_of_one_at_infty {f : α → β} [has_one β]
```

Comment:
Better `continuous.uniform_continuous_of_tendsto_cocompact`: the value of the limit at infinity has no bearing on the conclusion.